tests/integration/pilot/common/routing.go

```diff
@@ -692,12 +692,14 @@ metadata:
   name: default
 spec:
   hosts:
-    - {{ ( index .dstSvcs 0) }}
+{{- range $idx, $svc := .dstSvcs }}
+    - {{ $svc.Name }}.{{ $svc.Namespace }}
+{{- end }}
   http:
   - route:
 {{- range $idx, $svc := .dstSvcs }}
     - destination:
-        host: {{ $svc }}
+        host: {{ $svc.Name }}.{{ $svc.Namespace }}
```

Comment:
I didn't think  `{{ $svc.Name }}.{{ $svc.Namespace }}` was a valid destination...?

Reply:
Reverted to just service name.